<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeederControl</class>
 <widget class="QWidget" name="FeederControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Feeder</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Paramètre</string>
   </property>
   <widget class="QLineEdit" name="feederLE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>601</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Url à feeder</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Actif</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Dernier résultat du feeder:</string>
   </property>
  </widget>
  <widget class="QLabel" name="result_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Jamais lancé</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
